manifests/profile/moku.pp

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::moku
+#
+# @example
+#   include nebula::profile::moku
+class nebula::profile::moku (
+  String $init_directory = '/etc/moku/init'
+) {
+
```

Comment:
should be responsible for making sure init_directory exists

Reply:
I've updated it to ensure that's a directory, but it won't create the parent directory if it doesn't exist. We can probably wait to handle that until we puppetize installing moku itself.